turbo/jsonrpc/eth_call.go

```diff
@@ -612,14 +611,14 @@ func (api *BaseAPI) getWitness(ctx context.Context, db kv.RoDB, blockNrOrHash rp
 
 	// define these keys as "updates", but we are not really updating anything, we just want to load them into the grid,
 	// so this is just to satisfy the current hex patricia trie api.
-	updates := commitment.NewUpdates(commitment.ModeDirect, api.dirs.Tmp, commitment.KeyToHexNibbleHash)
+	reads := commitment.NewReads(commitment.ModeDirect, api.dirs.Tmp, commitment.KeyToHexNibbleHash)
```

Comment:
i do not see any reason to introduce another type for reads, as well as define another callback for `ReadTouch`

Reply:
another type for reads still does not needed.